packages/spark-core/base/_inputs.scss

```diff
@@ -96,7 +96,7 @@ textarea {
 
 .sprk-b-InputContainer > .sprk-b-Label,
 .sprk-b-InputContainer > label {
-  margin-bottom: $space-s;
+  margin-bottom: 12px;
```

Comment:
Should we be doing this instead of using the spacing classes? (We are talking about this in person, but I already wrote it so I'm adding it)